src/scss/core/_grid.scss

```diff
@@ -1,4 +1,4 @@
-@media (min-width: map-get($grid-breakpoints, lg)) {
+@include media-breakpoint-up(lg) {
 	.row {
 		margin-right: -$grid-gutter-width-lg / 2;
 		margin-left: -$grid-gutter-width-lg / 2;
```

Comment:
I would rather prefer to have the additional 7.5px as padding of the container on mobile. Since it's an exception we should also incapsulate it into a media-query exclusive for the all breakpoints up to lg

Reply:
By default, with the gutter 15px, bootstrap will create these:
- container with padding horizontal 7.5px
- row with margin horizontal -7.5px
- col with padding horizontal 7.5px

So the container already have 7.5px padding.
That's why I directly put container to have padding 15px instead.